server/api/views.py

```diff
@@ -158,3 +158,19 @@ def get(self, request, *args, **kwargs):
             writer.writerow(row)
 
         return response
+
+class CategoryRemovalView(APIView):
```

Comment:
Instead let's make `TransactionCategoryListCreateView` accept `DELETE`:

```
def delete(self, request, *args, **kwargs):
        // todo delete code
        return Response(status=status.HTTP_204_NO_CONTENT)
```

also, consider using `get_object_or_404()` to reduce boilerplate and avoid explicitly returning `404`